ingestion/tests/cli_e2e/test_cli_db_base.py

```diff
@@ -195,8 +195,8 @@ def run_command(self, command: str = "ingest") -> str:
                 self.test_file_path,
             ]
             process_status = subprocess.Popen(args, stderr=subprocess.PIPE)
-            process_status.wait()
-            return process_status.stderr.read().decode("utf-8")
+            _, stderr = process_status.communicate()
```

Comment:
thanks @nahuelverdugo for the help here. We were having a deadlock during the profiler